tendrl/monitoring_integration/grafana/alert_dashboard.py

```diff
@@ -69,18 +68,18 @@ def set_alert(panel, thresholds, severity, resource_name, title):
 
 def get_panels(resource_rows):
 
-    new_resource_panel = []
+    new_resource_panels = []
     try:
         for row in resource_rows:
             panels = row["panels"]
             for panel in panels:
                 if panel["type"] == "graph":
-                    new_resource_panel.append(copy.deepcopy(panel))
+                    new_resource_panels.append(copy.deepcopy(panel))
     except (KeyError, AttributeError) as ex:
         logger.log("debug", NS.get("publisher_id", None),
                    {'message': "Error in retrieving resource "
-                   "rows (get_panel) " + str(ex)})
-    return new_resource_panel
+                   "rows (get_panels) " + str(ex)})
```

Comment:
This is function name getting in log? do we need this?

Reply:
we don't need this, anyway logger module already finding log message from which function and line number. I will remove this.